### ISSUE TITLE
Her vi snakket om?

### ISSUE THREAD

**Issue by klementj** [laquist/CommencalBikesBootstrap#5]
https://github.com/laquist/CommencalBikesBootstrap/blob/f94f1b59670c26534f873faa5750a6470b5d5994/Opgavel%C3%B8sning/src/model/Bike.js#L52

Som den er lavet er det ikke på selve instancenen, men på et JS object i Bike.instances objektet. Derfor også static.

**Comment by laquist** [comment]
Yep. Det var egentlig bare en comment til mig selv. Jeg glemte bare at slette den comment efter at jeg havde valgt at lave den sådan.
Jeg tænkte bare at man også kunne have lavet den anderledes, hvor man kørte den på instansen, som så ændrede sine egne properties, og derefter blev updated til Bike.instances.
